### PR TITLE
Loaded tweets correctly via HTTPS

### LINKED ISSUE #113: Load tweets via HTTPS
Load tweets via HTTPS
Use : https://github.com/fossasia/2018.fossasia.org/pull/32/commits/9f84dae03dbc7f7c09bd3489b588702b18642c56 for reference.

### LINKED ISSUE #131: Fix tweets coming through loklak
Fix are currently not working. Please check how it works on other FOSSASIA sites and fix it.

### PR DESCRIPTION
**Pull request by nzec**
* Fixed Loading of tweets **(Credits : @0x48piraj)**
* Loaded Tweets via https:// **(Credits : @0x48piraj)**
* Fixed all mixed content things
* Fixes #131 
* Fixes #113 

**Here is the live preview :** https://duttaditya18.github.io/2016.fossasia.org/

---

*Took me a long while to figure this out!* 😄. *And it fixed all the issue in this repository!*

### CODE REVIEW

**Comment by nzec** [comment]
@fossasia/web-developers @fossasia/website-admins @mariobehling @geekyd

**Review by NaviRocker — APPROVED** [review]
LGTM

**Review by ParthS007 — APPROVED** [review]
LGTM !

**Review by supunTE — APPROVED** [review]
Good!

File: css/theme-lava.css [hunk 1]
```diff
@@ -2297,3 +2297,15 @@ footer.classic .social-profiles {
   color: #333333;
   font-size: 20px;
 }
+.contact-tweets {
+  padding: 0;
+}
+.contact-tweets .vertical-align {
```
Comment (harshit98) [inline css/theme-lava.css]: Leave one line.

File: css/theme-lava.css [hunk 2]
```diff
@@ -2297,3 +2297,15 @@ footer.classic .social-profiles {
...
+  height: 100%;
+  width: 100%;
+}
+.contact-tweets .vertical-align .col-md-5 {
```
Comment (harshit98) [inline css/theme-lava.css]: Here also, leave one line.

File: css/theme-lava.css [hunk 3]
```diff
@@ -2297,3 +2297,15 @@ footer.classic .social-profiles {
...
+  position: absolute;
+  top: 15%;
+  left: 5%;
+}
```
Comment (harshit98) [inline css/theme-lava.css]: Leave one line.

File: index.html [hunk 4]
```diff
@@ -3739,26 +3739,26 @@ <h1>FOSSASIA 2015 was a rousing success, check out the highlights from last year
...
-						<div class="col-md-5 col-sm-6">
-							<i class="icon social_twitter"></i>
-              <!-- Parameters documentation at https://github.com/fossasia/fossasia-loklak-webtweets/blob/gh-pages/README.md -->
-							<div class="tweets-feed" data-count=3 data-from="fossasia">
-							</div>
-							<span class="text-white">Follow <a target="default" href="http://twitter.com/fossasia">@fossasia</a> for more updates</span>
-              <img class="icon social_rss" src="img/rss.png" alt="rss">
-              <div id="result"></div>
-						</div>
+		<section class="contact-tweets">
```
Comment (harshit98) [inline index.html]: Correct the indentation to maintain code quality.

File: css/theme-lava.css (hunk 1 above)
Comment (nzec) [reply css/theme-lava.css]: All right on it!

File: index.html [hunk 5]
```diff
@@ -3739,26 +3739,26 @@ <h1>FOSSASIA 2015 was a rousing success, check out the highlights from last year
...
+			<div class="container vertical-align">
+                <div class="row">
+                    <div class="col-md-5 col-sm-6">
+                      <br><br>
+                        <i class="icon social_twitter"></i>
+          <!-- Parameters documentation at https://github.com/fossasia/fossasia-loklak-webtweets/blob/gh-pages/README.md -->
```
Comment (harshit98) [inline index.html]: Comment has not been indented properly.

File: index.html [hunk 6]
```diff
@@ -3817,5 +3817,23 @@ <h1>FOSSASIA 2015 was a rousing success, check out the highlights from last year
         <script src="js/tweetsfetcher.js"></script>
         <script src="js/placeholders.min.js"></script>
         <script src="js/scripts.js"></script>
+        <script>
```
Comment (harshit98) [inline index.html]: No need to indent `script` section. This section looks good to me! 👍

**Review by Ploypiti — APPROVED** [review]

File: index.html [hunk 7]
```diff
@@ -3817,5 +3817,23 @@ <h1>FOSSASIA 2015 was a rousing success, check out the highlights from last year
...
+            $(document).ready(function () {
+                $('#map').addClass('scrolloff');                // set the mouse events to none when doc is ready
+
+                $('#overlay').on("mouseup",function(){          // lock it when mouse up
+                $('#map').addClass('scrolloff');
+            //somehow the mouseup event doesn't get call...
+                });
+                $('#overlay').on("mousedown",function(){        // when mouse down, set the mouse events free
+                $('#map').removeClass('scrolloff');
```
Comment (harshit98) [inline index.html]: Better to remove the comments. It is clearly understood.

**Comment by nzec** [comment]
@harshit98 Done all the necessary changes!

File: index.html [hunk 8]
```diff
@@ -3741,81 +3738,95 @@ <h1>FOSSASIA 2015 was a rousing success, check out the highlights from last year
...
-        <script src="js/jquery.countdown.min.js"></script>
-        <script src="js/lightbox.min.js"></script>
-        <script src="js/smooth-scroll.min.js"></script>
-        <script src="js/tweetsfetcher.js"></script>
-        <script src="js/placeholders.min.js"></script>
-        <script src="js/scripts.js"></script>
+				</footer>
+			</div>
+
+			<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js" ></script>
```
Comment (harshit98) [inline index.html]: Indent this script properly.

File: index.html [hunk 9]
```diff
@@ -3741,81 +3738,95 @@ <h1>FOSSASIA 2015 was a rousing success, check out the highlights from last year
...
+	        <script src="js/flexslider.min.js"></script>
+	        <script src="js/jquery.plugin.min.js"></script>
+	        <script src="js/jquery.countdown.min.js"></script>
+	        <script src="js/lightbox.min.js"></script>
+	        <script src="js/smooth-scroll.min.js"></script>
+	        <script src="js/tweetsfetcher.js"></script>
+	        <script src="js/placeholders.min.js"></script>
+	        <script src="js/scripts.js"></script>
+	        <script>
+            $(document).ready(function () {
```
Comment (harshit98) [inline index.html]: Good work @duttaditya18. This looks great!

File: index.html [hunk 10]
```diff
@@ -3741,81 +3738,95 @@ <h1>FOSSASIA 2015 was a rousing success, check out the highlights from last year
...
-					<div class="row">
-						<div class="col-md-5 col-sm-6">
-							<i class="icon social_twitter"></i>
-              <!-- Parameters documentation at https://github.com/fossasia/fossasia-loklak-webtweets/blob/gh-pages/README.md -->
-							<div class="tweets-feed" data-count=3 data-from="fossasia">
-							</div>
-							<span class="text-white">Follow <a target="default" href="http://twitter.com/fossasia">@fossasia</a> for more updates</span>
-              <img class="icon social_rss" src="img/rss.png" alt="rss">
-              <div id="result"></div>
+	                <div class="row">
```
Comment (harshit98) [inline index.html]: Indentation of this block from here...

File: index.html (hunk 8 above)
Comment (nzec) [reply index.html]: I believe it is correctly indented.

File: index.html [hunk 11]
```diff
@@ -3741,81 +3738,95 @@ <h1>FOSSASIA 2015 was a rousing success, check out the highlights from last year
...
+	                      <br><br>
+	                        <i class="icon social_twitter"></i>
+	          				<!-- Parameters documentation at https://github.com/fossasia/fossasia-loklak-webtweets/blob/gh-pages/README.md -->
+	                        <div class="tweets-feed" data-count=3 data-query="fossasia">
+	                        </div>
+	                        <br><span class="text-white">Follow <a target="default" href="https://twitter.com/fossasia"><span class="text-white">@fossasia</span></a> for more updates</span>
+	                       <img class="icon social_rss" src="img/rss.png" alt="rss">
+	                       <div id="result"></div>
+	                    </div>
+						<div class="map-holder col-md-6 col-sm-4">
```
Comment (harshit98) [inline index.html]: To here... can be improved.

File: css/theme-lava.css (hunk 3 above)
Comment (nzec) [reply css/theme-lava.css]: Done

**Comment by nzec** [comment]
@harshit98 Done all the necessary Changes!

**Review by harshit98 — APPROVED** [review]

**Comment by nzec** [comment]
@harshit98 Thanks!

**Pull request merged**